packages/twenty-website/src/app/_components/user-guide/UserGuideCard.tsx

```diff
@@ -9,27 +9,38 @@ const StyledContainer = styled.div`
   color: ${Theme.border.color.plain};
   border: 2px solid ${Theme.border.color.plain};
   border-radius: ${Theme.border.radius.md};
-  padding: ${Theme.spacing(6)};
   gap: ${Theme.spacing(4)};
   display: flex;
   flex-direction: column;
   cursor: pointer;
-  width: 348px;
+  width: 340px;
+
+  @media (max-width: 385px) {
+    width: 280px;
+  }
 `;
 
 const StyledHeading = styled.div`
   font-size: ${Theme.font.size.lg};
   color: ${Theme.text.color.primary};
+  padding: 0 16px;
+  font-weight: ${Theme.font.weight.medium};
+  @media (max-width: 800px) {
+    font-size: ${Theme.font.size.base};
+  }
 `;
 
 const StyledSubHeading = styled.div`
-  font-size: ${Theme.font.size.sm};
+  font-size: ${Theme.font.size.xs};
   color: ${Theme.text.color.secondary};
   font-family: ${Theme.font.family};
+  padding: 0 16px 24px;
+  font-weight: ${Theme.font.weight.regular};
+  line-height: 21px;
 `;
 
 const StyledImage = styled.img`
-  width: 300px;
+  border-bottom: 1.13px solid var(--Transparency-Strong, #14141429);
```

Comment:
1.13px feels strange?what is var(--Transparency-Strong)? Can you introduce variables in the theme instead?